syft/pkg/cataloger/rust/test-fixtures/image-audit/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "hello_world"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+argh = "0.1"
```

Comment:
When I saw this name in test fixtures, I thought it was made up, but it's real: https://crates.io/crates/argh

What a great name for an arg parsing lib 😄

Reply:
right?!